pages/construct-libraries/[id].js

```diff
@@ -0,0 +1,450 @@
+                </>
+              )}
+              {constructLibrary.aliases?.length > 0 && (
+                <>
+                  <DataItemLabel>Aliases</DataItemLabel>
+                  <DataItemValue>
+                    <AliasList aliases={constructLibrary.aliases} />
+                  </DataItemValue>
+                </>
+              )}
+              <DataItemLabel>Summary</DataItemLabel>
+              <DataItemValue>{constructLibrary.summary}</DataItemValue>
+              <DataItemLabel>Scope</DataItemLabel>
+              <DataItemValue>{constructLibrary.scope}</DataItemValue>
+              <DataItemLabel>Selection Criteria</DataItemLabel>
+              <DataItemValue>
+                {constructLibrary.selection_criteria.join(", ")}
+              </DataItemValue>
+              {associatedDiseases.length > 0 && (
+                <>
+                  <DataItemLabel>Associated Diseases</DataItemLabel>
+                  <DataItemValue>
+                    <SeparatedList>
+                      {associatedDiseases.map((disease) => (
+                        <Link href={disease["@id"]} key={disease["@id"]}>
+                          {disease.term_name}
+                        </Link>
+                      ))}
+                    </SeparatedList>
+                  </DataItemValue>
+                </>
+              )}
+              {displayableTargetedGeneList.length > 0 && (
+                <>
+                  <DataItemLabel className="flex items-baseline">
+                    Targeted Genes
+                    <DataItemValueExpandButton
+                      isExpandable={isTargetedGenesExpandable}
+                      isExpanded={isTargetedGenesExpanded}
+                      onClick={setIsTargetedGenesExpanded}
+                    />
+                  </DataItemLabel>
+                  <DataItemValue className="flex items-center">
+                    <SeparatedList>
+                      {displayableTargetedGeneList.map((gene) => (
+                        <Link href={gene["@id"]} key={gene["@id"]}>
+                          {gene.geneid}
+                        </Link>
+                      ))}
+                    </SeparatedList>
+                    {isTargetedGenesTruncated && (
+                      <Icon.EllipsisHorizontal className="h-1 ml-1" />
+                    )}
+                  </DataItemValue>
+                </>
+              )}
+              {displayableTargetedLoci.length > 0 && (
+                <>
+                  <DataItemLabel className="flex items-baseline">
+                    Targeted Loci
+                    <DataItemValueExpandButton
+                      isExpandable={isTargetedLociExpandable}
+                      isExpanded={isTargetedLociExpanded}
+                      onClick={setIsTargetedLociExpanded}
+                    />
+                  </DataItemLabel>
+                  <DataItemValue>
```

Comment:
Do you mean this <DataItemValue> component can expand and collapse? I am not sure how?

Reply:
It appears collapsed by default:
<img width="716" alt="Screenshot 2023-08-11 at 16 19 32" src="https://github.com/IGVF-DACC/igvf-ui/assets/1181324/c4842aca-c4fc-4c3a-8199-a5f34f6acc4c">

When you click the icon after the label, it expands:
<img width="716" alt="Screenshot 2023-08-11 at 16 20 07" src="https://github.com/IGVF-DACC/igvf-ui/assets/1181324/9b2cd6c0-12c6-4fcc-b48f-166063818bf4">

Reply:
Thank you!